README.md

```diff
@@ -0,0 +1,35 @@
+# opentracing-compat
```

Comment:
I've not come across the term `compat` before, so I'm just going to assume this is the definition: 
<img width="596" alt="screenshot_598" src="https://user-images.githubusercontent.com/4147/60518236-56690600-9c9e-11e9-982f-73e7bfb0ee21.png">

Reply:
Whoops, did I assume the term was more common than it is? At least I'm not the only one: https://github.com/search?utf8=%E2%9C%93&q=compat&type=

E.g. https://github.com/developit/preact-compat

Reply:
No, it's definitely in use out there, I did find others, but this came up first in my search and amused me :)